docs/content/sre/multi-tenant-operator/changelog.md

```diff
@@ -1,26 +1,51 @@
 # Changelog
 
+## v0.8.x
+
+**v0.8.1**
+
+- fix: Updated release pipelines
+
+**v0.8.0**
+
+- feat: Allow custom roles for each tenant via label selector, more details in [custom roles document](./usecases/custom-roles.md)
```

Comment:
Add fix: Fix panic in ResourceSupervisor sleep functionality due to sending on closed channel